tools/mbedtls.cmake

```diff
@@ -52,15 +52,14 @@ if(NOT TARGET MbedTLS2_mbedtls)
     add_library(MbedTLS2::interface ALIAS MbedTLS2_interface)
 endif()
 
-set(MBEDTLS_3_VERSION 3.2.1)
+set(MBEDTLS_3_VERSION 3.4.0)
 
 FetchContent_Declare(
     mbedtls_3
-    GIT_REPOSITORY "https://github.com/ARMmbed/mbedtls"
+    GIT_REPOSITORY "https://github.com/Mbed-TLS/mbedtls"
     GIT_TAG v${MBEDTLS_3_VERSION}
     PATCH_COMMAND
-        ${CMAKE_CURRENT_LIST_DIR}/mbedtls_configure.sh <SOURCE_DIR> mbedtls_config.h &&
-        ${CMAKE_CURRENT_LIST_DIR}/mbedtls_patch.sh <SOURCE_DIR> ${CMAKE_CURRENT_LIST_DIR}/0001-Fix-missing-prototype-warning-when-MBEDTLS_DEPRECATE.patch
```

Comment:
If we do not need patch, can we remove these files? Or are these used somewhere?

Reply:
Updated the PR to remove the patch file as well.